### PR TITLE
Add CapacityDisplay component and TypeScript error fixes for Table Management files

### PR DESCRIPTION
**Pull request by rogers-ch**
I had to merge changes from v2 into my capacityComponent branch to get some updated files from v2, and git tagged a number of the v2 files as new files in my later commits. When I made my last commit, the automated checks made formatting changes to several files that I didn't actually work on. I'm not sure how to address this, so I thought I'd open a PR and let you look at the changes.

### CODE REVIEW

**Review by ralph-dev — APPROVED** [review]
LGTM, this is just Prettier Cleaning up the Code :) 
In the future try to write code cleanly without prettier

File: src/Containers/Chair.tsx [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
      */
-    position: Position,
+    position: Position;
     /**
      * Boolean value for whether someone is seated in the chair
      * True if someone is seated in the chair, otherwise false
      */
-    isSeated?: boolean,
-
+    isSeated?: boolean;
```
Comment (Ampatte2) [inline src/Containers/Chair.tsx:13]: This is not used by the component.

File: src/Containers/CircleTable.tsx [hunk 2]
```diff
@@ -1,59 +1,54 @@
...
     /**
      * The occupancy status for the table
      */
-    occupancyStatus: occupancyStatusTypes,
+    occupancyStatus: occupancyStatusTypes;
     /**
      * The seating/reservation time for the party at the table
      */
-    reservationTime: Date,
+    reservationTime?: Date;
```
Comment (Ampatte2) [inline src/Containers/CircleTable.tsx:24]: Same with this variable not used by the component

**Review by Ampatte2 — APPROVED** [review]
There are two props not used by the components, otherwise looks good!

LGTM

**Pull request merged**